cluster-autoscaler/cloudprovider/aws/ec2_instance_types/gen.go

```diff
@@ -83,7 +83,14 @@ func main() {
 	flag.Parse()
 	defer klog.Flush()
 
-	instanceTypes, err := aws.GenerateEC2InstanceTypes(*region)
+	sess, err := session.NewSession(&aws.Config{
```

Comment:
This file doesn't import session so `make generate` now fails with:

```
go generate ./cloudprovider/aws
# command-line-arguments
ec2_instance_types/gen.go:86:15: undefined: session
ec2_instance_types/gen.go:86:39: undefined: aws.Config
ec2_instance_types/gen.go:87:15: undefined: aws.String
cloudprovider/aws/aws_manager.go:17: running "go": exit status 2
```

Reply:
@gjtempleton The AWS Session doesn't work without region (maybe it worked some day I am not sure), I did some rework on gen.go if you think I should put it in different PR I can do that?